tests/Phinx/Migration/ManagerTest.php

```diff
@@ -6034,7 +6036,7 @@ public function testMigrationWithDropColumnAndForeignKeyAndIndex()
         $adapter->disconnect();
 
         $this->manager->setConfig($config);
-        $this->manager->migrate('production', '20190928205056');
+        $this->manager->migrate('production', 20190928205056);
```

Comment:
Some tests were passing  string as a version instead of an int. This appears to only affect tests as the migration loading code converts to int.